packages/cli/src/telemetry/index.ts

```diff
@@ -122,8 +126,10 @@ export class Telemetry {
 		clearInterval(this.pulseIntervalReference);
 		void this.track('User instance stopped');
 		return new Promise<void>((resolve) => {
-			if (this.client) {
-				this.client.flush(resolve);
+			if (this.postHog) this.postHog.shutdown();
```

Comment:
nitpick: over time i've come to prefer consistency in blocks, wdyt?
```suggestion
			if (this.postHog) {
				this.postHog.shutdown();
			}
```

Reply:
I find one-liners more readable but not precious about it. Changed.